meson.build

```diff
@@ -36,6 +36,23 @@ if get_option('b_coverage')
 	add_project_arguments(compiler.get_supported_arguments(['-fstack-protector-all', '--param=ssp-buffer-size=4']), language: 'cpp')
 endif
 
+# howardhinnant/date has several names - look for them, from the most
+# to the least explicit name.
+# In Meson 0.60.0, this can be replaced with a simpler:
+#
+#     dependency('howardhinnant-date', 'hinnant-date', 'date')
+#
+date_dep = dependency('howard-hinnant-date', required: false)
+if not date_dep.found()
+	date_dep = dependency('howardhinnant-date', required: false)
+endif
+if not date_dep.found()
+	date_dep = dependency('hinnant-date', required: false)
+endif
+if not date_dep.found()
+	date_dep = dependency('date', fallback: ['hinnant-date', 'date_dep'])
+endif
+
```

Comment:
Why is this here? It's the same code as below, and it's only needed if there's no working stdlib's C++20 chrono library.

Reply:
Quite right on both counts... I updated.